bin/cask

```diff
@@ -153,20 +166,36 @@ def format_version(version):
     return '.'.join(str(part) for part in version)
 
 
+def get_missing_packages():
+    """Get a report of the missing direct dependencies.
+
+    The report is a human-readable list of the any direct dependencies which are
+    required but not present. The empty string is returned if there are none.
+
+    """
+    process = subprocess.Popen([sys.executable, CASK, "missing"], stdout=subprocess.PIPE)
```

Comment:
Return code, yes, will add.

Returning a structured string or a boolean, I thought about that. The problem is that we can get the same error from inside python (cask exec, cask emacs) or inside Emacs (cask outdated, eval and several others). Seems to me that the error message should look the same in both cases. Otherwise, I have to return a parsable list from cask missing, then print it to look the same as the error message that we would get from lisp.